lib/modules/manager/maven-wrapper/artifacts.spec.ts

```diff
@@ -0,0 +1,309 @@
+import type { Stats } from 'fs';
+import os from 'os';
+import type { StatusResult } from 'simple-git';
+import { join } from 'upath';
+import { envMock, mockExecAll } from '../../../../test/exec-util';
+import { env, fs, git, mockedFunction, partial } from '../../../../test/util';
+import { GlobalConfig } from '../../../config/global';
+import { resetPrefetchedImages } from '../../../util/exec/docker';
+import { getPkgReleases } from '../../datasource';
+import { updateArtifacts } from '.';
+
+jest.mock('../../../util/fs');
+jest.mock('../../../util/git');
+jest.spyOn(os, 'platform').mockImplementation(() => 'darwin');
+jest.mock('../../../util/exec/env');
+jest.mock('../../datasource');
+
```

Comment:
```suggestion
process.env.CONTAINERBASE = 'true';

```

https://github.com/renovatebot/renovate/blob/ff648e687df564449241fb39716c5a514905b55f/lib/modules/manager/bundler/artifacts.spec.ts#L34

this shold fix the missing install command for `binarySource=install`